scripts/basic-workflow-tests.sh

```diff
@@ -34,7 +34,7 @@ SCRIPTDIR=$(dirname "$0")
 export E2E_TEST_DIR
 # E2E_TEST_DIR="$(pwd)/__e2e_test"
 export CONFIG_HOME
-export DATA_HOME
+export CACHE_HOME
```

Comment:
Did you mean to remove `DATA_HOME`? It seems like you renamed all instances of `DATA_HOME` to now be `CACHE_HOME`.